playground/infrastructure/helper.py

```diff
@@ -13,14 +13,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import re
 import os
+import yaml
 
+from dataclasses import dataclass
 from typing import List
-from api.v1.api_pb2 import Sdk, SDK_UNSPECIFIED, SDK_JAVA, Status, STATUS_UNSPECIFIED
+from api.v1.api_pb2 import Sdk, SDK_JAVA, SDK_UNSPECIFIED, STATUS_UNSPECIFIED, SDK_GO, SDK_PYTHON, SDK_SCIO
 
-SUPPORTED_SDK = {'java': SDK_JAVA}
-PATTERN = re.compile('Beam-playground:\n {2} *name: \w+\n {2} *description: .+\n {2} *multifile: (true|false)\n {2} *categories:\n( {4} *- [\w\-]+\n)+')
+SUPPORTED_SDK = {'java': SDK_JAVA, 'go': SDK_GO, 'py': SDK_PYTHON, "scala": SDK_SCIO}
+END_OF_LICENCE = "limitations under the License."
+START_OF_IMPORT = "\nimport "
```

Comment:
Does it work for every SDK? I think in Python imports may also look like `from x import y`. Let's design this in our Examples Deployment doc. And please add an assumption that we have imports in all examples we parse.

Reply:
Updated logic to find tag into the file. Now it supports `import ...` and `from ... import ...`.

Reply:
Thanks for updating the doc! Just want to double-confirm, do `"\nimport "` and `"\nfrom "` cover all cases for our supported SDKs? Java, Python, Go, Scala?

Reply:
Changed logic of searching.